### PR TITLE
Integrate Jenkins 2.7.1 LTS (jenkins-dcos v2.0.0)

### PR DESCRIPTION
**Pull request by ssk2**
Merge after #85.

### CODE REVIEW

**Comment by rji** [comment]
This will land in version 2.0 of our Jenkins integration for DC/OS. Let's hold off on merging it for the time being.

**Comment by rji** [comment]
Seems to be working as expected on DC/OS 1.8-dev. @mesosphere/velocity-team, ready for review!

File: Dockerfile [hunk 1]
```diff
@@ -39,11 +39,11 @@ java ${JVM_OPTS}                                    \
     -Dhudson.udp=-1                                 \
     -Djava.awt.headless=true                        \
     -Dhudson.DNSMultiCast.disabled=true             \
+    -Djenkins.install.runSetupWizard=false          \
```
Comment (colin-msphere) [inline Dockerfile]: This works as expected.

**Comment by colin-msphere** [comment]
Do we want to call out the new Blue Ocean UI in the README? Or mention that the 2.x setup wizard is being skipped?

Other than those two questions: 👍

**Comment by rji** [comment]
I'm not sure if we want to do that in the README or in a blog post / documentation. For what it's worth, the blueocean-* plugins are mentioned in the README. @ssk2?

**Comment by ssk2** [comment]
@rji blog post!

**Pull request merged**